tensorflow/core/grappler/optimizers/remapper.cc

```diff
@@ -4451,6 +4460,9 @@ Status Remapper::Optimize(Cluster* cluster, const GrapplerItem& item,
       ctx.graph_view.SortTopologically(/*ignore_cycles=*/false, {}));
 
   const int num_nodes = item.graph.node_size();
+  const int intra_op_parallelism_threads =
```

Comment:
We're getting a warning about this not being used anywhere.  Is that intentional?

Reply:
Hi @cantonios, `intra_op_parallelism_threads` is used here https://github.com/tensorflow/tensorflow/blob/e57d550a06b891cad890b2a963ef33b37c4a23ec/tensorflow/core/grappler/optimizers/remapper.cc#L4607 when deciding whether to rewrite Sigmoid+Mul to `_MklSwish`. That part of the code is under `#ifdef DNNL_AARCH64_USE_ACL` so I believe that maybe putting around that line the same `ifdef` should help with the warning as for now it is only used there or alternatively the line can go inside of that `ifdef`?

Reply:
Glad @cantonios checked, because I just removed it. XD Adding it back now near line 4607 where it's used. Thank you, @milpuz01!

Reply:
Okay, looks like @penpornk moved it internally.  Thanks.

Reply:
Thanks for adding that